CSS/style.css

```diff
@@ -0,0 +1,199 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@200;300;400;600&display=swap');
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    height: 100vh;
+    overflow-y: scroll;
+    overflow-x: hidden;
+}
+
+.navbar {
+    width: 100%;
+    position: absolute;
+    height: 72px;
+    left: 0;
+    top: 0;
+    display: flex;
+    justify-content: space-between;
+    padding: 26px 164px 26px 155px;
+    align-items: center;
+    background: #fff;
+}
```

Comment:
- [ ] Good job setting up everything :100: , however you should stick as much as possible to the Figma design, that means the design you did should be as much as possible as the one use in the FIGMA design, and some property should also be the same :100: 

### Your design

![Screenshot from 2023-02-17 02-25-52](https://user-images.githubusercontent.com/80458199/219519414-1f198172-81bb-4eb1-b3da-0d7c32c2290d.png)


### Figma design

![Screenshot from 2023-02-17 02-26-45](https://user-images.githubusercontent.com/80458199/219519388-271ca05b-e940-44b6-a194-f32348f9721d.png)